app/controllers/answers_controller.rb

```diff
@@ -5,8 +5,9 @@ def create
     @question = Question.find(params[:answer][:question])
     @answer = @question.answers.new(answer_params)
     @answer.user = current_user
+    @comment = @question.comments.new
     if @answer.save
-      flash[:success] = "You commented!"
+      flash[:success] = "You answered the question!"
```

Comment:
Great job getting all of this logic built out. That form was a jerk with the amount of parameters that it needed.